src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/benchmarks.wcfcore.Perf.yml

```diff
@@ -125,7 +102,7 @@ profiles:
       load:
         endpoints: 
           - http://asp-citrine-db:5001
-  
+
   aspnet-citrine-arm:
     variables:
       serverPort: 5000
```

Comment:
This is were the 8231 value should be I assume.

Reply:
The ‘serverPort: 8231’ in the scenarios is not used, so I removed it. [cb04d79](https://github.com/dotnet/wcf/pull/4394/commits/cb04d796e5cf53e0f94211fb2370382039678ffc)